examples/gcs/microgcs.py

```diff
@@ -12,14 +12,26 @@
 #Set up option parsing to get connection string
 import argparse  
 parser = argparse.ArgumentParser(description='Tracks GPS position of your computer (Linux only). Connects to SITL on local PC by default.')
-parser.add_argument('--connect', default='127.0.0.1:14550',
+parser.add_argument('--connect',
                    help="vehicle connection target. Default '127.0.0.1:14550'")
```

Comment:
Default in help string should be removed. In fact just copy paste exactly the same startup from the other examples.